configs/serviceops/types.yml

```diff
@@ -0,0 +1,4 @@
+types:
+- name: global/types/GenericListCheckerType
```

Comment:
We need to decide what we're trying to namespace here.

My understanding is that the namespace is about scoping the use of different pieces of config state. In that context, then I think we would end up with just global/<foo> and <service_name>/bar for the types, handlers, and actions. For the templates, I think we'll end up with istio/<foo> for the built-in stuff and company_name/<foo> for custom templates provided by vendors.